fastly_data_adapter.go

```diff
@@ -0,0 +1,91 @@
+package statsig
```

Comment:
nit: can we name this like `statsig-fastly-adapter` or something more generic like `main`? Naming it `statsig` doesn't really make sense.